mongo_orchestration/common.py

```diff
@@ -48,6 +49,7 @@ class BaseModel(object):
         {'role': 'dbAdminAnyDatabase', 'db': 'admin'},
         {'role': 'readWriteAnyDatabase', 'db': 'admin'}
     ]
+    socket_timeout = DEFAULT_SOCKET_TIMEOUT
```

Comment:
The intention is that `socket_timeout` is consistent across Servers, ReplicaSets, and ShardedClusters. I'm not sure that there's a need to be able to set this property separately on each, and it's configured globally from the command-line anyway.